addons/arrays/test_shuffle.sqf

```diff
@@ -21,18 +21,14 @@ TEST_OP(count _result,==,count _original,_fn);
     TEST_OP(_x,in,_original,_fn);
 } forEach _result;
 
-// Test depecated version.
+// test modify original
```

Comment:
It has to be removed, since the behavior of the function has changed. Otherwise there will always be an error.